src/tools/schema2graph.ts

```diff
@@ -20,14 +20,8 @@ export class SchemaNode {
   // 'Particle_Handle' names that need to be type aliased to it.
   name: string;
   aliases: string[] = [];
-
-  // We are working to move the Kotlin entity classes to be within a particle class. This
-  // provides a unique namesapce, and thus we can have the entity class named just Handle.
-  // to make this easier, we have the kotlinName and kotlinAliases that use just Handle
-  // as the name.
-  // TODO(heimlich): Update to use schema names for kotlin where possible.
-  kotlinName: string;
-  kotlinAliases: string[] = [];
+  particleName: string;
+  connections: string[] = [];
```

Comment:
Probably worth updating the comment here.

Reply:
Updated!